cadctap/setup.py

```diff
@@ -55,6 +56,9 @@ def readme():
 for entry_point in entry_point_list:
     entry_points['console_scripts'].append('{0} = {1}'.format(entry_point[0],
                                                               entry_point[1]))
+# copy example_query.sql file to /tmp so that the examples in cadc-tap query -h work
```

Comment:
I don't think it's worth doing this when most of the time is not going to be used.

Reply:
Removed.